lesson-3/script.js

```diff
@@ -0,0 +1,89 @@
+function userPortrait () {
+
+    let dateUser;
+    
+    function trueDate () {
+        return !dateUser || dateUser < 1900 || dateUser > 2005;
+    }
+
+    do {
+        dateUser = prompt('Enter please your birth date');
+
+        if (trueDate(dateUser)) {
+            alert('It is a pity that you did not want to enter your birth date...');
+        }
+    } while (trueDate(dateUser));
+
+
+    let currentDate = new Date().getFullYear() - dateUser;
+
+    let cityUser;
+
+    function trueCity () {
+        return !cityUser || cityUser == ' ';
+    }
+
+    do {
+        cityUser = prompt('What city do you live in?');
+
+        if (trueCity(cityUser)) {
+            alert('It is a pity that you did not want to enter your city...');
+        }
+    } while (trueCity(cityUser));
+
+    let toLowerCaseCity = cityUser.toLowerCase();
+    let otherCity;
+    let capitalCity;
+    
+    switch (toLowerCaseCity) {
+        case 'kiev','kyiv':
```

Comment:
'kiev' will not work here, if you want 2 cases with one result use:
```suggestion
        case 'kiev':
        case 'kiyv':
```